docs/celsius-converter-option/index.md

```diff
@@ -258,76 +160,22 @@ string to float. Hint: Use `Js.Float.isNaN`.
 <b>1.</b> To prevent a space from producing 32.00 degrees Fahrenheit, just add a
 call to `String.trim` in your render logic:
 
-<!--#prelude#
-let celsius = "1";
-let convert = x => x;
-let _ =
--->
-```reason
-{(
-  String.trim(celsius) == ""
-    ? {js|? °F|js}
-    : (
-      switch (celsius |> float_of_string_opt |> Option.map(convert)) {
-      | None => "error"
-      | Some(fahrenheit) when fahrenheit > 212.0 => {js|Unreasonably hot🥵|js}
-      | Some(fahrenheit) =>
-        Js.Float.toFixedWithPrecision(fahrenheit, ~digits=2)
-        ++ {js| °F|js}
-      }
-    )
-)
-|> React.string}
-```
+<<< Snippets.re#string-trim{1-2}
```

Comment:
This highlights the second line too, with the `ºF` part, not sure if intended

<img width="751" alt="image" src="https://github.com/melange-re/melange-for-react-devs/assets/220424/ecf7f9f9-c80a-4966-9489-57833860f0e4">

Reply:
This was intentional, to emphasize that both lines are new additions to the original expression